xml/security_ldap_install.xml

```diff
@@ -429,6 +429,30 @@ binddn = cn=Directory Manager
      </para>
     </callout>
    </calloutlist>
-  </example>
+  </example>  
+       <warning>
+       <title>New negation feature in sudoers.ldap</title>
+       <para>
+         If you are using sudoers.ldap, instead of the traditional 
+         <command>sudo</command> command, one of its limitations is 
```

Comment:
I think this should sudoers file not sudo command here. sudoers.ldap replaces/supplements the sudoers / sudoers.d files.